build/depvers.sh

```diff
@@ -56,9 +58,6 @@ for (( i=0; i < ${#pkginfo[@]}; i+=2 )); do
   if test "${git}" -eq 1; then
     vers=$(git -C "${dir}" rev-parse HEAD 2>/dev/null )
   else
-    # file ownership sometimes doesn't work in docker and hg gets grumpy about
-    # this file, which is basically empty anyway, so just delete it
-    rm "${dir}/.hg/hgrc"
```

Comment:
See
https://github.com/cockroachdb/cockroach/commit/efc8a57e557c1929093135ebcf1585e738ee83d1
On Jun 28, 2015 12:11 PM, "Ben Darnell" <notifications@github.com> wrote:

> In build/depvers.sh
> <https://github.com/cockroachdb/cockroach/pull/1528#discussion_r33424144>:
>
> > @@ -56,9 +58,6 @@ for (( i=0; i < ${#pkginfo[@]}; i+=2 )); do
> >    if test "${git}" -eq 1; then
> >      vers=$(git -C "${dir}" rev-parse HEAD 2>/dev/null )
> >    else
> > -    # file ownership sometimes doesn't work in docker and hg gets grumpy about
> > -    # this file, which is basically empty anyway, so just delete it
> > -    rm "${dir}/.hg/hgrc"
>
> What has changed since this was added that makes it no longer necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1528/files#r33424144>.
>